### PR TITLE
Update Kibana integration to collect task manager HTTP metrics

### PR DESCRIPTION
**Pull request by ymao1**
## What does this PR do?

Updates the Kibana HTTP integration to collect task manager SLI metrics from the Kibana API endpoint added in [this PR](https://github.com/elastic/kibana/pull/163652).

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## How to test this PR locally

- Run the stack using `elastic-package`: `elastic-package stack up --version=8.10.0-SNAPSHOT -v -d`
- Build the package:
  - `cd integrations/packages/kibana`
  - `elastic-package build`
- Deploy the package: `elastic-package stack up -v -d --services package-registry`
- Navigate to https://localhost:5601/app/integrations/browse
- Select and add the Kibana integration
- Verify you see the new `HTTP metrics > Kibana task manager metrics` integration
- Configure the integration for host `https://kibana:5601` and username/password `elastic:changeme`
- Verify you see a datastream for `metrics-kibana.task_manager_metrics-default`
- Verify you see data in the datastream (`GET metrics-kibana.task_manager_metrics-default/_search`) with the `kibana.task_manager_metrics.metrics` field populated

## Related issues

-

### CODE REVIEW

File: packages/kibana/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.3.7"
```
Comment (klacabane) [inline packages/kibana/changelog.yml:2]: Since we add feature can we bump minor instead ?

**Review by klacabane — APPROVED** [review]
LGTM

File: packages/kibana/changelog.yml (hunk 1 above)
Comment (ymao1) [reply packages/kibana/changelog.yml:2]: So `2.4.0`?

Comment (ymao1) [reply packages/kibana/changelog.yml]: Updated to `2.4.0` in this commit: https://github.com/elastic/integrations/pull/7205/commits/8e970138506e1ab404ccef4d2a7752e08fca0d16

**Pull request merged**